src/pages/caseDetail/caseDetailView.js

```diff
@@ -7,6 +7,7 @@ import { CustomDataTable } from 'bento-components';
 import TableFooter from '@material-ui/core/TableFooter';
 import TableRow from '@material-ui/core/TableRow';
 import TablePagination from '@material-ui/core/TablePagination';
+import { useDispatch } from 'react-redux';
 import StatsView from '../../components/Stats/pageSpecificStatsController';
```

Comment:
Here is the pageSpecificStatsController

Reply:
If you refresh in caseDetail page, you'll see stats bar won't load and a spinner spins forever. I added the code to solve this issue.